dll/win32/shimgvw/shimgvw.c

```diff
@@ -507,7 +507,7 @@ pLoadImageFromNode(SHIMGVW_FILENODE *node, HWND hwnd)
 {
     WCHAR szTitleBuf[800];
     WCHAR szResStr[512];
-    LPWSTR pchFileTitle;
+    LPWSTR pchFileTitle = NULL;
```

Comment:
Is this change really needed? As far as I can see there is no access to it before the assignment via PathFindFileNameW().
And https://learn.microsoft.com/en-us/windows/win32/api/shlwapi/nf-shlwapi-pathfindfilenamew
seems to promise to write something there in all flow-cases.

Reply:
This isn’t necessary, fix it.